README.md

```diff
@@ -213,6 +213,7 @@ interface Renderers<TRow, TSummaryRow> {
   renderCheckbox?: Maybe<(props: RenderCheckboxProps) => ReactNode>;
   renderRow?: Maybe<(key: Key, props: RenderRowProps<TRow, TSummaryRow>) => ReactNode>;
   renderSortStatus?: Maybe<(props: RenderSortStatusProps) => ReactNode>;
+  renderCell?: Maybe<(key: Key, props: CellRendererProps<TRow, TSummaryRow>) => ReactNode>;
```

Comment:
```suggestion
  renderCell?: Maybe<(key: Key, props: CellRenderProps<TRow, TSummaryRow>) => ReactNode>;
```

Maybe we can rename now

Reply:
It conflicts with existing `RenderCellProps` used for `renderCell` property of `Column`